src/gravel/api/services.py

```diff
@@ -149,6 +149,20 @@ async def create_service(
     return CreateResponse(success=True)
 
 
+@router.delete("/{service_name}",
+               name="Delete service by name")
+async def delete(service_name: str, request: Request):
```

Comment:
The code to secure access via authentication is missing.
```suggestion
async def delete(service_name: str, request: Request, _=Depends(jwt_auth_scheme)):
```